### PR TITLE
update configuration for FD-323

### PR DESCRIPTION
**Pull request by fuzhe1989**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

https://github.com/pingcap/tics/pull/3762

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by LiSong0214 — APPROVED** [review]

**Review by LiSong0214 — APPROVED** [review]

**Comment by TomShawn** [comment]
> 

@fuzhe1989 Please sign the CLA. Thanks!
/cc @shichun-0415

**Review by windtalker — APPROVED** [review]
LGTM

File: tiflash/tiflash-configuration.md [hunk 1]
```diff
@@ -155,6 +155,9 @@ delta_index_cache_size = 0
     cop_pool_size = 0
     ## 从 v5.0 引入，表示 TiFlash Coprocessor 最多同时执行的 batch 请求数量。如果请求数量超过了该配置指定的值，多出的请求会排队等待。如果设为 0 或不设置，则使用默认值，即物理核数的两倍。
     batch_cop_pool_size = 0
+    ## 从 v5.4 引入，表示是否启用可自动扩展的线程池，这项功能可以显著提高 TiFlash 在高并发场景的 CPU 利用率。默认为 false。
```
Comment (TomShawn) [inline tiflash/tiflash-configuration.md:158]: @fuzhe1989 Compatibility change list 说 FD-323 还新增了 profiles.default.enable_elastic_threadpool
profiles.default.elastic_threadpool_init_cap 和 
profiles.default.elastic_threadpool_shrink_period_ms 线程池控制参数，默认关闭（实验特性）。这里要写进文档里吗？

Comment (fuzhe1989) [reply tiflash/tiflash-configuration.md:158]: 后面两个参数暂时不希望用户调节，我理解应该可以不写进文档吧？

Comment (TomShawn) [reply tiflash/tiflash-configuration.md:158]:
```suggestion
    ## 从 v5.4.0 引入，表示是否启用可自动扩展的线程池，这项功能可以显著提高 TiFlash 在高并发场景的 CPU 利用率。默认为 false。该功能为实验特性，不推荐在生产环境中开启。
```

File: tiflash/tiflash-configuration.md [hunk 2]
```diff
@@ -155,6 +155,9 @@ delta_index_cache_size = 0
     cop_pool_size = 0
     ## 从 v5.0 引入，表示 TiFlash Coprocessor 最多同时执行的 batch 请求数量。如果请求数量超过了该配置指定的值，多出的请求会排队等待。如果设为 0 或不设置，则使用默认值，即物理核数的两倍。
     batch_cop_pool_size = 0
+    ## 从 v5.4.0 引入，表示是否启用可自动扩展的线程池，这项功能可以显著提高 TiFlash 在高并发场景的 CPU 利用率。默认为 false。该功能为实验特性，不推荐在生产环境中开启。
+    # enable_elastic_threadpool = 0
```
Comment (shichun-0415) [inline tiflash/tiflash-configuration.md:159]: 上面是“默认为 false”，下面是“XX=0”，是否需要修改一致？

Comment (fuzhe1989) [inline tiflash/tiflash-configuration.md:159]:
```suggestion
    # enable_elastic_threadpool = false
```

Comment (fuzhe1989) [reply tiflash/tiflash-configuration.md]: done

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Review by qiancai — APPROVED** [review]

**Pull request merged**